### ISSUE TITLE
Support ItemGroups / [System.IO.Directory]::GetDirectories

### ISSUE THREAD

**Issue by meastp** [Caphyon/clang-power-tools#392]
This is (part of) my `.vcxproj` file:

```xml
<ItemGroup>
  <AllExportedIncludeFolders Include="$([System.IO.Directory]::GetDirectories(&quot;c:\myproject\src\&quot;, &quot;include&quot;, SearchOption.AllDirectories))" />
</ItemGroup>
<PropertyGroup>
  <IncludePath>@(AllExportedIncludeFolders);$(IncludePath);</IncludePath>
</PropertyGroup>
```
The source tree contains multiple `include` folders (one for each project), and the `ItemGroup` is supposed to collect these paths and add them to `IncludePath`.

When building, I get:
```
VERBOSE: SET_VAR IncludePath: @(AllExportedIncludeFolders);C:\qms\src;;
```
And the header files in the include folders are not found.
The reason might be that `ItemGroup` is not supported, or that `[System.IO.Directory]::GetDirectories` (within `ItemGroup`) is not supported.

**Comment by meastp** [comment]
This is still an issue with the latest version 4.3.0 of Clang Power Tools.

**Comment by cpp-red-lion** [comment]
@hero101111 is investigating this.  
To my knowledge, this isn't yet fixed (don't know why...)

**Comment by hero101111** [comment]
Hello,

We've implemented support for this in Clang Power Tools `4.5`, which is fresh out the release oven.

Best regards,
Gabriel.

**Comment by meastp** [comment]
I can confirm that this now works. Thank you :)
